### PR TITLE
Disable coroutine empterpretify for Wasm Backend

### PR DESCRIPTION
**Pull request by aheejin**

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -7307,6 +7307,7 @@ def do_test_coroutine(self, additional_settings):
   def test_coroutine_asyncify(self):
     self.do_test_coroutine({'ASYNCIFY': 1})
 
+  @no_wasm_backend('EMTERPRETIFY causes JSOptimizer to run, which is disallowed')
```
Comment (dschuff) [inline tests/test_core.py]: how about "unsupported" rather than "disallowed" ?

Comment (dschuff) [reply tests/test_core.py]: or better yet, "Unsupported with the wasm backend" to make it more clear why

Comment (aheejin) [reply tests/test_core.py]: Done. I copied the line from another test, so I fixed that too.

**Comment by aheejin** [comment]
Unless you are waiting for the CI, you can merge it (I don't have the access)

**Pull request merged**